feat(docs): add XRPL EVM Sidechain Axelar docs (#7)
ripple/docs.xrplevm.org: docs/evm-sidechain/axelar-overview.md

Currently, the transfer of XRP from the XRP Ledger (XRPL) to the XRPL EVM Sidechain is enabled through the Axelar Network. Axelar is a decentralized cross-chain communication protocol designed to provide secure and scalable connectivity across Web3 ecosystems. It leverages a proof-of-stake (PoS) consensus model, which has been proven robust in multiple blockchain environments such as Ethereum, Polygon, and Cosmos. This consensus mechanism is maintained by a network of validators that ensure message integrity and prevent double-spending or malicious activities during interchain transactions. By abstracting the complexities of cross-chain interactions, Axelar enables the development of "Interchain dApps" that natively support multi-chain functionalities, thus broadening the scope and capabilities of decentralized applications beyond single-chain constraints.

> ## Intergration summary

Reviewer: You can remove this section header.

> <embed src="/snippets/_axelar-overview-disclaimer.md" />

Reviewer: Move this to the top of the page.

> The integration of the Axelar Network as a bridge for native token transfers between the XRPL Ledger and the XRPL EVM Sidechain blockchains enables seamless interoperability across these distinct blockchain environments. Axelar functions as a cross-chain communication layer, facilitating secure, permissionless transfers of native assets without the need for intermediaries or wrapped tokens.

Reviewer: We can remove this paragraph, since it's repeating what was said in the paragraph above. Also I don't think that last line about transferring assets without intermediaries or wrapped tokens is inaccurate?

Using Axelar is still creating a wrapped version of the token when bridging, isn't it? And Axelar itself is an intermediary.

> By leveraging Axelar's decentralised network of validators and robust message-passing protocol, assets can move between the XRPL's native ledger and its EVM-compatible sidechain with full transactional consistency. This integration allows for expanded use cases, such as decentralised finance (DeFi) applications and cross-chain liquidity solutions, by bridging the gap between the XRPL's high-throughput, low-cost transactions and the programmable smart contract capabilities of the EVM ecosystem.

Reviewer: ```suggestion
Leveraging Axelar's decentralised network of validators and robust message-passing protocol enables assets to move between the XRPL's native ledger and its EVM-compatible sidechain with full transactional consistency. This integration combines the XRPL's high-throughput and low-cost transactions with the programmable smart contract capabilities of the EVM ecosystem, allowing for expanded use cases, such as decentralised finance (DeFi) applications and cross-chain liquidity solutions.
```
